src/platform/packages/shared/kbn-ftr-common-functional-services/services/es.ts

```diff
@@ -25,4 +33,114 @@ export function EsProvider({ getService }: FtrProviderContext): Client {
           authOverride: systemIndicesSuperuser,
         }
   );
+
+  const refreshInterval = config.get('indexRefreshInterval');
+  // do not optimize for cloud runs or if explicitly disabled
+  if (!!process.env.TEST_CLOUD || refreshInterval === false) {
```

Comment:
Are we going to get the 50ms refresh interval even when running tests locally, too? It looks like the `!!process.env.TEST_CLOUD` check may be preventing it. I think it's important we keep local and CI environments as close as possible. What do you think?